test/execution/compiler_test.cpp

```diff
@@ -68,6 +68,68 @@ class CompilerTest : public SqlBasedTest {
   static constexpr vm::ExecutionMode MODE = vm::ExecutionMode::Interpret;
 };
 
+// NOLINTNEXTLINE
+// TEST_F(CompilerTest, SimplerSeqScanTest) {
```

Comment:
Is there a reason this test is commented out?

Reply:
We originally wanted to test the implementation without thread local state argument but got some problem with the test case. We'll delete this part in the next commit

Reply:
@YuhongZhang98 Is this test case no longer valid?